doc/api/console.markdown

```diff
@@ -85,15 +94,48 @@ Example:
     console.timeEnd('100-elements');
     // prints 100-elements: 262ms
 
-## console.trace(message[, ...])
+### console.trace(message[, ...])
 
 Print to stderr `'Trace :'`, followed by the formatted message and stack trace
 to the current position.
 
-## console.assert(value[, message][, ...])
+### console.assert(value[, message][, ...])
 
 Similar to [assert.ok()][], but the error message is formatted as
 `util.format(message...)`.
 
+## Class: Console
+
+<!--type=class-->
+
+Use `require('console').Console` or `console.Console` to access this class.
+
+    var Console = require('console').Console;
+    var Console = console.Console;
+
+You can use `Console` class to custom simple logger like `console`, but have
+different output streams.
```

Comment:
"You can use the `Console` class to create a custom logger like `console`, but with different output streams.